cmd/bootstrap/daemon-up.sh

```diff
@@ -19,15 +19,24 @@ fi;
 # Pull the latest inertia daemon.
 sudo docker pull $IMAGE_REPOSITORY
 
-# Run container with access to the host docker socket - this
-# is necessary because we want the daemon to be able start
+# Make Project directory
+mkdir $HOME/project
+
+# Run container with access to the host docker socket and related executables -
+# this is necessary because we want the daemon to be able start
 # and stop containers on the host. It's also controversial,
 # https://www.lvh.io/posts/dont-expose-the-docker-socket-not-even-to-a-container.html
 # It's also recommended,
 # https://jpetazzo.github.io/2015/09/03/do-not-use-docker-in-docker-for-ci/
-# As a result, this container has root access on the remove vps.
+# As a result, this container has root access on the remote vps.
 sudo docker run -d \
     -p "$PORT":8081 \
     -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /usr/bin/docker:/usr/bin/docker \
```

Comment:
Looks like the container should be the same as the host:
> Docker originally used LinuX Containers (LXC), but later switched to runC (formerly known as libcontainer), which runs in the same operating system as its host. This allows it to share a lot of the host operating system resources.

[source](https://stackoverflow.com/a/16048358)

Reply:
The first comment to that thread:

> Ken, in some places you conflate the OS with the kernel. All containers on a host run under the same kernel, but the rest of the OS files can be unique per container.

One host kernel is orchestrating our container processes here. But each container we run via the host may use a different OS. Here is the example in inertia's case (say we've just allocated a gcloud vps with ubuntu installed on it):

1. We boostrap the VPS (installing docker via [this](https://get.docker.com/)).
2. It figures out what operating system its running on and installs `docker` for that OS.
3. You mount that binary into the daemon (running Alpine Linux).

Theres no guarantee that the binary is executable on Alpine Linux! 🙈 

Heres a working example (pretending the host is running macOS :joy:):

```bash
~/git/tmp  master ✔
▶ cp `which docker` .
~/git/tmp  master ✗                                                                                                                                                              
▶ docker run -it -v `pwd`:/home alpine
/ $ ls
bin    dev    etc    home   lib    media  mnt    proc   root   run    sbin   srv    sys    tmp    usr    var
/ $ cd home
/home $ ls
docker
/home $ ./docker
./docker: line 1: ������: not found
./docker: line 2: @��__gosymtab__TEXT������__gopclntab__TEXT���5�A���__cstring__TEXT�t�l�t�__const__TEXTP: not found
./docker: line 2: ��P: not found
./docker: line 2: �__unwind_info__TEXT
                                      ���
                                         ��__eh_frame__TEXT��-��h__DATA��: not found
./docker: line 2: ���,: not found
./docker: line 2: �H__PAGEZER__TEXT����
                                       __text__TEXT#n�#�__stubs__TEXTn��n�__stub_helper__TEXTD�D��__rodata__TEXT�__typelink__TEXT: not found
./docker: line 3: ���: not found
./docker: line 4: syntax error: unexpected word (expecting ")")
/home $
```

Reply:
Ohhhh I see

Reply:
Hang on, I've been mounting my local (macOS) installation into the daemon, should that not be working?

Reply:
You don't appear to be using the docker binary anywhere in this PR. That's probably why it works. 

The go docker library does everything you need, it never has to access a docker binary in a file somewhere, all it needs is to look at `/var/run/docker.sock` and find _a_ socket there 😄 - which we provide with `-v /var/run/docker.sock:/var/run/docker.sock`.

I think we can just remove the `-v /usr/bin/docker:/usr/bin/docker` here. It's not doing anything and it would break if we tried to actually use it somewhere.

Reply:
Oh yeah you're right, was using the docker binary for back when I was still using the shell script - not needed anymore

Speaking of which running that `project-up.sh` script was working before and I think that used the mounted docker binary?

Reply:
Not sure! It may work sporadically, if we happen to use a host compatible with Alpine - but in general we can't assume that a binary built from source in one OS will work inside other OS.